src/language/sv.json

```diff
@@ -57,26 +55,30 @@
     "new-yritys": "Nytt företag?",
     "yritys": "Företag",
     "yritys-laatijat": "Upprättare",
-    "ohjeet": "Laatijan ohjeet SV?",
-
+    "ohjeet": "Upprättarens anvisningar",
     "confirmation": {
-      "title": "Varmistus poistumisesta SV",
-      "description": "Lomakkeella on tallentamattomia muutoksia. Menetät kaikki tallentamattomat muutokset, jos poistut sivulta. SV",
-      "message": "Oletko varma, että haluat poistua sivulta? SV",
-      "forward": "Poistu sivulta SV",
-      "cancel": "Peruuta SV"
+      "title": "Bekräftelse om att lämna",
+      "description": "Formuläret innehåller osparade ändringar. Du mister alla osparade ändringar om du lämnar sidan.  ",
+      "message": "Är du säker på att du vill lämna sidan?  ",
+      "forward": "Lämna sidan",
+      "cancel": "Avbryt"
+    },
+    "etusivu": "Etusivu SV?",
+    "valvonta": {
+      "valvonta": "Valvonta SV?",
+      "oikeellisuus": "Oikeellisuuden valvonta SV?"
     }
   },
   "errors": {
     "load-error": "Sökningen av uppgifterna misslyckades.",
-    "unauthorized": "Olet kirjautunut ulos järjestelmästä. Voit kirjautua uudestaan toisella välilehdellä, jos tällä välilehdellä on jotakin kesken. SV?",
-    "forbidden": "Puutteelliset käyttöoikeudet SV?",
-    "unavailable": "Järjestelmä on väliaikaisesti pois käytöstä. Yritä hetken kuluttua uudestaan. SV?",
-    "whoami": "Käyttäjätietojen hakemisessa tapahtui odottamaton virhe. Uloskirjautuminen voi korjata ongelman. SV?",
-    "row-error": "Virhe rivillä SV?",
-    "file-read-error": "Tiedoston lukemisessa tapahtui odottamaton virhe. SV?"
+    "unauthorized": "Du har loggat ut ur systemet. Du kan logga in igen på en annan flik, om du har något oavslutat på den här fliken.  ",
+    "forbidden": "Bristfälliga användningsrättigheter",
+    "unavailable": "Systemet är tillfälligt ur bruk. Försök på nytt om en stund.  ",
+    "whoami": "Ett oväntat fel inträffade vid sökning av användaruppgifterna. Att logga ut kan lösa problemet.  ",
+    "row-error": "Fel på rad",
+    "file-read-error": "Ett oväntat fel inträffade vid läsning av filen.  "
   },
-  "units": { "days": "dagar", "year": "år", "pieces": "St", "celsius": "°C" },
+  "units": { "year": "år", "pieces": "St", "celsius": "°C", "days": "vrk SV?" },
```

Comment:
Meneekö tässä `days` väärään suuntaan?

Reply:
Teknisesti ei mene väärään suuntaan. Käännöstoimistolle ei ole lähetetty versiota jossa tuota daysiä on ollut mukana. Se on siis lisätty meidän toimesta käännöksineen kaikkineen sinne sv.jsonin puolelle. Luonnollisesti koko sv.json korvautuu uusien käännösten myötä, koska ei näitä kukaan täysjärkinen ala purkamaan käsin. Nyt tosin pitää varmaan tehdä joku scripti mikä mergeää tuon vanhan ja uuden sv.json käännöstiedoston ja fillata uusiksi.